web/satellite/src/components/team/ProjectMemberListItem.vue

```diff
@@ -0,0 +1,135 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+<template>
+    <div class="user-container">
+        <div class="user-container__base-info">
+            <div class="checkbox" >
+            </div>
+            <div class="user-container__base-info__avatar" :style="avatarData.style">
+                <h1>{{avatarData.letter}}</h1>
+            </div>
+            <p class="user-container__base-info__user-name">{{this.itemData.formattedFullName()}}</p>
+        </div>
+        <p class="user-container__date">{{this.itemData.joinedAtLocal()}}</p>
```

Comment:
why not computed value? Separate logic from view

Reply:
We have styles, not only letter.